src/slc/mailrouter/browser/views.py

```diff
@@ -2,7 +2,7 @@
 import sys
 from tempfile import NamedTemporaryFile
 import traceback
-from StringIO import StringIO
+from six import StringIO
```

Comment:
This should be from io import BytesIO

Reply:
The appropriate class to store text is io.StringIO, while the appropriate class to store bytes is io.BytesIO.
I might be tricked because I saw in the code:
```
self.request.stdin.read().encode("utf-8")
```
so I thought we were dealing with bytes.
Anyway if six.StringIO can be replaced by a more specific io class it is better